src/components/LandingPage.tsx

```diff
@@ -0,0 +1,443 @@
+import GameData from "../assets/game-data.png"
+import RandomGames from "../assets/random-games.jpg"
+import CiriLooking from "../assets/ciri-looking.png"
+import GodOfWar from "../assets/god-of-war-cover.jpg"
+import SantaMonica from "../assets/santa-monica-logo.png"
+import Insomniac from "../assets/insomniac-logo.png"
+import Ubisoft from "../assets/ubisoft-logo.png"
+import BandaiNamco from "../assets/bandai-namco-logo.png"
+import NaughtyDog from "../assets/naughty-dog-logo.png"
+
+const features = [
+  {
+    name: 'All Platforms',
+    description: 'Stay on top of the platforms each game was released on. Play whichever you prefer.',
+    icon: GiRetroController,
+  },
+  {
+    name: 'All Game Genres',
+    description: 'Heard of a game genre you didn\'t know about? Stay on top of all game genres!',
+    icon: GiBroadsword,
+  },
+  {
+    name: 'Miss The Classics?',
+    description: 'Don\'t know your favorite classic as well as you\`d like? Studio, release date, platforms, all in one place.',
+    icon: GiBrickWall,
+  },
+  {
+    name: 'Game Ratings',
+    description: 'Do you like to know the game notes? Get lists of the best and worst titles based on Metacritic ratings.',
+    icon: GiStarsStack,
+  }
+]
+const metrics = [
+  { id: 1, stat: '2M+', emphasis: 'Game Screenshots', rest: 'for you to see the brilliance of the game environment and characters.' },
+  { id: 2, stat: '1M+', emphasis: 'Ratings', rest: 'by the community and review specialists given by Metacritic.' },
+  { id: 3, stat: '45K+', emphasis: 'Publishers', rest: 'like Sony, CD Projekt Red, Bethesda and much more!' },
+  { id: 4, stat: '58K+', emphasis: 'Tags', rest: 'including game genres, platforms and etc.' },
+]
+const footerNavigation = {
+  solutions: [
+    { name: 'Marketing', href: '#' },
+    { name: 'Analytics', href: '#' },
+    { name: 'Commerce', href: '#' },
+    { name: 'Insights', href: '#' },
+  ],
+  support: [
+    { name: 'Pricing', href: '#' },
+    { name: 'Documentation', href: '#' },
+    { name: 'Guides', href: '#' },
+    { name: 'API Status', href: '#' },
+  ],
+  company: [
+    { name: 'About', href: '#' },
+    { name: 'Blog', href: '#' },
+    { name: 'Jobs', href: '#' },
+    { name: 'Press', href: '#' },
+    { name: 'Partners', href: '#' },
+  ],
+  legal: [
+    { name: 'Claim', href: '#' },
+    { name: 'Privacy', href: '#' },
+    { name: 'Terms', href: '#' },
+  ],
+  social: [
+    {
+      name: 'Facebook',
+      href: '#',
+      icon: (props:any) => (
+        <svg fill="currentColor" viewBox="0 0 24 24" {...props}>
+          <path
+            fillRule="evenodd"
+            d="M22 12c0-5.523-4.477-10-10-10S2 6.477 2 12c0 4.991 3.657 9.128 8.438 9.878v-6.987h-2.54V12h2.54V9.797c0-2.506 1.492-3.89 3.777-3.89 1.094 0 2.238.195 2.238.195v2.46h-1.26c-1.243 0-1.63.771-1.63 1.562V12h2.773l-.443 2.89h-2.33v6.988C18.343 21.128 22 16.991 22 12z"
+            clipRule="evenodd"
+          />
+        </svg>
```

Comment:
I think it's better to put this svg in another file and just import on this component. Make the code more readable.